sandbox/src/mocks/stu3/changeShortlistAndSendForTriage/responses/MinimalRequest.json

```diff
@@ -72,6 +88,10 @@
                                     "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReviewReason-1",
                                     "code": "TRIAGE"
                                 }
+                            },
+                            {
+                                "url": "comments",
+                                "valueString": "We are in need of a Triage service"
```

Comment:
I can't get this to appear in a response - is it possible to add comments when creating a triage request? Would this need to be provided in the request, which it isn't?

Reply:
I have reverted this change back. I can't find how to add the comment (as part of running a test), although I believe you can add it as part of the changing of the shortlist part, not so much the creation of the triage request itself. As I can't verify this though, I rather delete it. Will create a tech debt to verify the schema itself, as it does state this field should appear if it is available.